sentry-android-ndk/src/main/java/io/sentry/android/ndk/IModuleListLoader.java

```diff
@@ -0,0 +1,10 @@
+package io.sentry.android.ndk;
+
+import io.sentry.protocol.DebugImage;
+
+/** Used only for making the Module list loader testable */
+interface IModuleListLoader {
+  DebugImage[] getModuleList();
+
+  void clearModuleList();
```

Comment:
`h`: Why do we need this method? I see only the tests calling it and also `IDebugImagesLoader.clearDebugImages()`. This seems like an NDK specific thing to me, because it has a cache. I think `NativeModuleListLoader` should take care of clearing the cache if necessary.

Reply:
this is necessary, the debug list is loaded on SDK init (or 1st event, still trying out) and it reads from the process (in memory).
if an App loads a native library after the SDK init, the cached list would be out of date and one would need to clean the list by themselves, we cant detect this automatically.

Reply:
Hm, I get the idea. So our users would need to call this manually? This could then get a bit tricky I guess. Maybe then caching is not the best idea here?

Reply:
yes, they'd need to do it manually only if they need to (most o the cases, they don't).
why it's not a good idea? where/how would you cache it?

Reply:
As discussed: Because JNI calls are expensive we want to avoid loading the `debugImages` from `sentry-native` often. So the cache makes sense.